build-vm

```diff
@@ -783,7 +783,10 @@ vm_setup() {
     if test -n "$VM_USE_MKFS_COPYIN" ; then
 	echo "Using mkfs copyin mode to populate the root filesystem"
 	mkdir_build_root
-        rm -rf "$BUILD_ROOT"/* "$BUILD_ROOT"/.[^.]* 
+	if test ! -w /root ; then
+	    find "$BUILD_ROOT" -type d -not -perm /200 -print0 | xargs -0 --no-run-if-empty chmod +w
+	fi
+	find "$BUILD_ROOT" -mindepth 1 -maxdepth 1 -print0 | xargs -0 --no-run-if-empty rm -rf
```

Comment:
not sure why this is needed, it works for me without.. ?

Reply:
if you build the same package twice, reusing the build root you get stuff like
rm: cannot remove '/var/tmp/build-root/usr/lib64/librpmbuild.so.9.2.0': Permission denied
This is because in TW several OS directories are mode 555